doc/changelog/04-tactics/17503-redexpr.rst

```diff
@@ -1,6 +1,6 @@
 - **Added:**
-  :tacn:`lazy`, :tacn:`simpl` and :tacn:`cbn` and the associated :cmd:`Eval` and :tacn:`eval` reductions
-  learnt to do head reduction when given flag `head`
+  :tacn:`lazy`, :tacn:`simpl`, :tacn:`cbn` and :tacn:`cbv` and the associated :cmd:`Eval`
+  and :tacn:`eval` reductions learnt to do head reduction when given flag `head`
```

Comment:
```suggestion
  The :tacn:`lazy`, :tacn:`simpl`, :tacn:`cbn` and :tacn:`cbv` tactics and the associated :cmd:`Eval`
  and :tacn:`eval` reductions will now do head reduction when the `head` flag is specified
```

"learned" is preferred in American English.

Reply:
We don't prefer american english AFAIK

Reply:
On this point, I don't know what are the current conventions, but there was at least a decision long time ago to adopt American English in the reference manual and in the code.

Reply:
The convention remains American English, a decision I was not involved in.

Reply:
Enforcing this convention seems mostly useless except to annoy contributors. Is the decision documented anywhere? If not let's just forget it.

Reply:
You prefer annoying users who read the documentation?

Reply:
Of course. The reader's annoyance is based on spelling prescriptivism or some such nonsense, the contributor's annoyance is based on having to do work for no benefit.